shared/bazel/rules/objectivec_rules.bzl

```diff
@@ -0,0 +1,29 @@
+load("@rules_cc//cc:defs.bzl", "objc_library")
+
+OBJC_COMPILER_FLAGS = [
+    "-stdlib=libc++",
+    "-fPIC",
+]
+
+def wpilib_objc_library(
+        name,
+        srcs = [],
+        deps = [],
+        copts = None,
+        is_cpp = True,
+        include_arc = True,
+        **kwargs):
+    copts = copts or []
```

Comment:
Same, comment for this macro would be good.  What is arc, and why should I care?

Reply:
Arc is something specific to objective C, its Automatic Reference Counting. Theres likely going to be a lot of terms in here that are complex, because objective c has some quirks.

Reply:
Understood.  Even just "Automatic Reference Counting" attached to the acronym in a comment is helpful for folks following along behind trying to understand and help update.

Reply:
Added a docstring to the function